qos_iommu.adoc

```diff
@@ -1,9 +1,10 @@
 [[QOS_IOMMU]]
 == IOMMU Extension for QoS ID
 
-A method to associate QoS IDs with requests to access resources by the IOMMU, as
-well as with devices governed by it, is required for effective monitoring and
-allocation. This section specifies a RISC-V IOMMU cite:[IOMMU] extension to:
+A method to associate QoS IDs with requests to access resources by the Input-Output 
+Memory Management Unit (IOMMU), as well as with devices governed by it, is required 
+for effective monitoring and allocation. This section specifies a RISC-V I
```

Comment:
Will it render right if I and OMMU have a line break?

Reply:
I'll check

Reply:
Works fine for me